packages/build/tests/core/snapshots/tests.js.md

```diff
@@ -2249,3 +2249,105 @@ Generated by [AVA](https://avajs.dev).
+
+## Successfully builds ES module function with feature flag
+
+> Snapshot 1
+
+    `␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build                                                 ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    > Version␊
+      @netlify/build 1.0.0␊
+    ␊
+    > Flags␊
+      debug: true␊
+      featureFlags:␊
+        - buildbot_es_modules_esbuild␊
+      repositoryRoot: packages/build/tests/core/fixtures/functions_es_modules␊
+      testOpts:␊
+        pluginsListUrl: test␊
+        silentLingeringProcesses: true␊
+    ␊
+    > Current directory␊
+      packages/build/tests/core/fixtures/functions_es_modules␊
+    ␊
+    > Config file␊
+      packages/build/tests/core/fixtures/functions_es_modules/netlify.toml␊
+    ␊
+    > Resolved config␊
+      build:␊
+        publish: packages/build/tests/core/fixtures/functions_es_modules␊
+        publishOrigin: default␊
+      functionsDirectory: packages/build/tests/core/fixtures/functions_es_modules/functions␊
+    ␊
+    > Context␊
+      production␊
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      1. Functions bundling                                         ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    Packaging Functions from functions directory:␊
+     - function_with_es_module.js␊
+    ␊
+    ␊
+    (Functions bundling completed in 1ms)␊
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build Complete                                        ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    (Netlify Build completed in 1ms)`
+
+## Fails build for ES module function if feature flag is off
+
+> Snapshot 1
+
+    `␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build                                                 ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    > Version␊
+      @netlify/build 1.0.0␊
+    ␊
+    > Flags␊
+      debug: true␊
+      repositoryRoot: packages/build/tests/core/fixtures/functions_es_modules␊
+      testOpts:␊
+        pluginsListUrl: test␊
+        silentLingeringProcesses: true␊
+    ␊
+    > Current directory␊
+      packages/build/tests/core/fixtures/functions_es_modules␊
+    ␊
+    > Config file␊
+      packages/build/tests/core/fixtures/functions_es_modules/netlify.toml␊
+    ␊
+    > Resolved config␊
+      build:␊
+        publish: packages/build/tests/core/fixtures/functions_es_modules␊
+        publishOrigin: default␊
+      functionsDirectory: packages/build/tests/core/fixtures/functions_es_modules/functions␊
+    ␊
+    > Context␊
+      production␊
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      1. Functions bundling                                         ␊
```

Comment:
I'm a bit confused why this passes, because I confirmed that https://github.com/netlify/zip-it-and-ship-it/blob/9b0c6d56e4e05704122fdd55cacc81d75cf40aa1/src/runtimes/node/index.js#L20 is set to `false` and [`zisi`](https://github.com/netlify/zip-it-and-ship-it/blob/9b0c6d56e4e05704122fdd55cacc81d75cf40aa1/src/runtimes/node/index.js#L28) is being returned as the bundler